src/app/services/schema/app-schema.service.ts

```diff
@@ -544,10 +547,14 @@ export class AppSchemaService {
       }
 
       if (relation.operatorName === '=') {
-        this.handleEqualOperatorNameSubscription(payload, relation);
+        subscription.add(
+          timer(0).pipe(take(1)).subscribe(() => this.handleEqualOperatorNameSubscription(payload, relation)),
```

Comment:
I've noticed these `timer` calls, and I think there should be a better way to implement this. My suggestion is to try `detectChanges` or one of the `NgZone` methods

Reply:
tried your recommendations, didn't work for me.